### PR TITLE
Add Banhawy minneapolis md file

### LINKED ISSUE #18: Create a new file for Banhawy Minneapolis
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a PR
- [ ] Discuss
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by Banhawy**
closes #18

### CODE REVIEW

**Comment by James-Mulcahy** [comment]
All Your Base Are Belong To Us

File: Banhawy-Minneapolis.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Minneapolis, MN
+
+##Reccommendations
+- Eat at 5 Guys
```
Comment (James-Mulcahy) [inline Banhawy-Minneapolis.md:4]: Best burgers anywhere!

**Review by James-Mulcahy — COMMENTED** [review]
Delicioso

File: Banhawy-Minneapolis.md (hunk 1 above)
Comment (Banhawy) [reply Banhawy-Minneapolis.md:4]: And FRIES!

**Pull request merged**